spring/boot2-autoconfigure/src/main/java/com/linecorp/armeria/spring/DocServiceConfigurator.java

```diff
@@ -25,7 +25,7 @@
 import com.linecorp.armeria.server.docs.DocServiceBuilder;
 
 /**
- * Configures a {@link DocService} built by {@link ArmeriaAutoConfiguration}.
+ * Configures a {@link DocService} built by Armeria auto configuration.
```

Comment:
This class is copied to `boot2-webflux-autoconfigure` and it cannot connect to the class there.